src/igvfd/schemas/tabular_file.json

```diff
@@ -135,7 +137,9 @@
             ],
             "enum_descriptions": {
                 "barcode to element mapping": "A file describing the mapping of barcodes to genomic elements.",
+                "barcode to hashtag mapping": "A file describing the mapping of barcodes to hashtags",
                 "barcode to sample mapping": "A file describing the mapping of barcodes with to biological sample.",
```

Comment:
Not from your ticket, but just noticed this definition says "barcodes with to biological sample" -- it seems like the "with" is accidentally added there? Not sure we should fix it here though.